docs/developer.md

```diff
@@ -290,10 +304,38 @@ PASS
 ```
 
 To test the multinamespace setup, you can use
+
 ```
 ./run_operator_locally.sh --rebuild-operator
 ```
-It will automatically create an `acid-minimal-cluster` in the namespace `test`. Then you can for example check the Patroni logs:
+It will automatically create an `acid-minimal-cluster` in the namespace `test`.
+Then you can for example check the Patroni logs:
+
 ```
 kubectl logs acid-minimal-cluster-0
 ```
+
+## Introduce additional configuration parameters
+
```

Comment:
By convention if the same option is defined in the operator config and in the cluster manifest, the latter takes precedence

Reply:
Isn't this already documented somewhere? Thought it was [here](https://github.com/zalando/postgres-operator/blob/master/docs/reference/operator_parameters.md). I can add a link, nevertheless.